Cinema Journal/K.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constants.swift
+//  Cinema Journal
+//
+//  Created by Homer on 10/21/20.
+//
+
+import Foundation
+
+struct K {
```

Comment:
Better naming. Obj-C styled is global `let kAPIKey = "some"`. You're using more clear way, so why not naming it's like `Constants` or something else human readable